library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -251,15 +251,19 @@ internal class TransactionPayloadFragment :
                 val context = fragment.context ?: return false
                 context.contentResolver.openFileDescriptor(uri, "w")?.use {
                     FileOutputStream(it.fileDescriptor).use { fos ->
-                        when {
-                            type == TYPE_REQUEST -> {
-                                transaction.requestBody?.byteInputStream()?.copyTo(fos)
+                        when (type) {
+                            TYPE_REQUEST -> {
+                                transaction.requestBody?.byteInputStream()?.copyTo(fos) ?: throw IOException()
```

Comment:
(commenting once but you have 3 occurrences of this).

Mayne we can add a short message in the `IOException()`?

Reply:
Good idea.